client/extensions/woocommerce/state/sites/customers/selectors.js

```diff
@@ -1,86 +1,58 @@
-/**
- * External dependencies
- */
-
-import { filter, get } from 'lodash';
-
 /**
  * Internal dependencies
  */
 import { getSelectedSiteId } from 'state/ui/selectors';
 
+function inFlight( state, siteId, searchTerm ) {
+	return state?.extensions?.woocommerce?.sites?.[ siteId ]?.customers.isSearching[ searchTerm ];
+}
+
 /**
  * @param {object} state Whole Redux state tree
  * @param {string} searchTerm Search term to check
  * @param {number} [siteId] Site ID to check. If not provided, the Site ID selected in the UI will be used
  * @returns {boolean} Whether the search results for a given term have been successfully loaded from the server.
  */
-export const isCustomerSearchLoaded = (
-	state,
-	searchTerm,
-	siteId = getSelectedSiteId( state )
-) => {
-	const inFlight = get(
-		state,
-		`extensions.woocommerce.sites[${ siteId }].customers.isSearching[${ searchTerm }]`
-	);
+export function isCustomerSearchLoaded( state, searchTerm, siteId = getSelectedSiteId( state ) ) {
 	// Strict check because it could also be undefined.
-	return false === inFlight;
-};
+	return inFlight( state, siteId, searchTerm ) === false;
```

Comment:
Any reason to go away from Yoda-style conditions? FWIW, I'm not their biggest fan too.

Reply:
I'm not a huge fan either, but I think they make very little sense when the thing being tested isn't a valid lvalue. Since this changed from a variable (valid lvalue) to a function call (not a valid lvalue), I went ahead and made the switch. Happy to put it back, though 🙂

Reply:
I'm happy to pretend I never saw you changed them to non-Yoda 🙈